### DISCUSSION TITLE
Windows环境下实现任务栏暂停、播放切换音乐等操作

### DISCUSSION THREAD

**Discussion by e8xl** [SPlayer-Dev/SPlayer#189] [想法 Ideas]
![image](https://github.com/imsyy/SPlayer/assets/56244103/19e8505a-795e-445e-96db-2a41b1ba6e13)
**如图** 这样的暂停功能在不方便打开窗口的时候提供了很大的便利
搜索了一篇csdn的文章 里面提到了网易云开发者关于[taskbar](https://github.com/zhangjiwei1221/blog/blob/master/taskbar/)的处理
文章链接：https://blog.csdn.net/qq_41698074/article/details/134591316

**Comment by imsyy** [comment]
会在重构后支持该功能
